tests/integration/sharding_tests/helpers.py

```diff
@@ -13,9 +13,10 @@
     get_secret_data,
 )
 
-TIMEOUT = 10 * 60
+TIMEOUT = 15 * 60
```

Comment:
QQ my memory fails me. Were you ever able to find the reason for the application taking awhile to deploy in the tests?

Reply:
From what I've seen, we were on the verge of timing out when deploying full cluster with local images.
I guess using remote images pushed us beyond timeout.
There is a small overhead using the self hosted runners it seems which comes on top of that.
I'm not happy at all about this increased timeout but there's nothing much we can do for now…

Reply:
QQ Did you ask Carl about it? They are an expert when it comes to caching images on CI

Reply:
We talked about it and as it seems there's no big difference between GH runners and self hosted runners, it might just be that we really take too much time because we spend already 5-6 minutes waiting for machines to be available when deploy the cluster

Reply:
ack ty